addons/web_tour/static/src/js/tour_manager.js

```diff
@@ -52,21 +55,27 @@ var RunningTourActionHelper = core.Class.extend({
     },
     _click: function (values) {
         var href = values.$element.attr("href");
-        if (href && href.length && href[0] !== "#" && values.$element.is("a")) {
+        if (href && href.length && href[0] !== "#" && values.$element.is("a") && href !== 'javascript:void(0)') {
             window.location.href = href;
         } else {
-            values.$element.mousedown().mouseup().click();
+            values.$element.trigger("mouseenter").mousedown().mouseup().click().trigger("mouseleave");
         }
     },
     _text: function (values, text) {
         this._click(values);
 
         text = text || "Test";
-        if (values.consume_event === "input") {
-            values.$element.val(text).trigger("input");
+        if (values.$element.is('select')) {
```

Comment:
I would put this if section inside the else selection. So if (... === "input) {...} else { if (... is select) {} else {} }